src/main/java/org/mqttbee/mqtt/handler/publish/MqttOutgoingQoSHandler.java

```diff
@@ -155,13 +155,8 @@ private MqttPublishWrapper wrapPublish(
             isNewTopicAlias = false;
         } else {
             final MqttTopicImpl topic = publish.getTopic();
+            isNewTopicAlias = topicAliasMapping.createIfAbsent(topic, publish.getTopicAliasUsage());
```

Comment:
What I do not like about this approach: we have to perform always 2 HashMap.get operations although not necessary